### DISCUSSION TITLE
Is a polygon overlay possible?

### DISCUSSION THREAD

**Discussion by martin-ueding** [osmandapp/OsmAnd#18214] [Maps Related]
I play around with the explorer tiles. The idea is to have an activity in each tile, where a tile is defined by the usual Open Street Map web Mercator tiling at zoom level 14.

When I am on the go, I'd like to see which tiles I haven't got yet. For this I have build a collection of scripts which download the activity data from Strava and generates the missing tiles as polygons. When overlaid on a map, it looks like this:

![Screenshot_20230924_084930](https://github.com/osmandapp/OsmAnd/assets/976924/03eb113a-b6b0-491c-9fae-ae2119ec9c07)

I can export this in various formats. The issue is that OsmAnd doesn't support the GeoJSON that I've used here. It looks like this:

```json
{
  "features": [
    {
      "geometry": {
        "coordinates": [
          [
            [
              -6.28418,
              53.357109
            ],
            [
              -6.262207,
              53.357109
            ],
            [
              -6.262207,
              53.343993
            ],
            [
              -6.28418,
              53.343993
            ],
            [
              -6.28418,
              53.357109
            ]
          ]
        ],
        "type": "Polygon"
      },
      "properties": {},
      "type": "Feature"
    },
    …
  ]
}
```

The KML format also supports polygons, but OsmAnd doesn't support that either. The GPX format (which is supported) doesn't support polygons. For the time being I have simulated this with a bunch of square tracks, but I cannot fill/shade them this way.

Is there some other way to get these polygons into OsmAnd?

**Comment by mariush444** [comment]
Convertion to gpx (workaround ) or creating obf with this polygons.
